shell-ui/src/NavBar.js

```diff
@@ -118,7 +160,7 @@ export const Navbar = ({ options, userGroupsMapping }: { options: Options, userG
           userGroups,
         ),
         {
-          label: <Item icon={'fas fa-sign-out-alt'} label={'Log Out'} />,
+          label: <Item icon={'fas fa-sign-out-alt'} label={language === 'en' ? 'Log Out' : 'Déconnexion'} />,
```

Comment:
Shouldn't we setup i18n on the navbar as well? This is acceptable short term, but won't scale as we add translated text or supported languages.